react/src/lib/components/WebvizDialog/WebvizDialog.tsx

```diff
@@ -138,13 +138,13 @@ export const WebvizDialog: React.FC<WebvizDialogProps> = (props) => {
             document.getElementsByClassName("WebvizDialog--active")
         );
         activeDialogs.forEach((dialog) => {
-            if (dialog.id !== props.id) {
+            if (dialog !== dialogRef.current) {
                 dialog.classList.remove("WebvizDialog--active");
             }
         });
         if (
             dialogRef.current &&
-            !dialogRef.current?.classList.contains("WebvizDialog--active")
+            !dialogRef.current.classList.contains("WebvizDialog--active")
```

Comment:
Shorter
```typescript
if (!dialogRef.current?.classList.contains("WebvizDialog--active")) {
```

Reply:
I have to correct this one, I missed the `!`. For a one-liner, use:
```typescript
if (dialogRef.current?.classList.contains("WebvizDialog--active") === false) {
```

else, keep it as is.